main/signers/lattice/adapter.ts

```diff
@@ -70,7 +70,7 @@ export default class LatticeAdapter extends SignerAdapter {
           reloadAddresses = reloadAddresses || (lattice.addresses.length < lattice.accountLimit)
           needsUpdate = true
         }
-
+        //@ts-expect-error -- baseUrl was made private in gridplus-sdk@v1.2.4
```

Comment:
is there any other way to detect if the base URL has changed?

Reply:
@mholtzman we're pushing an update that will make `baseUrl` _public_ again.

Reply:
[Commit](https://github.com/floating/frame/pull/845/commits/f3493308dfe58669392ea6078dabf9f32ff33384).